docs/resources/client/models/bakedmodel.md

```diff
@@ -0,0 +1,117 @@
+# Baked Models
+
+`BakedModel`s are the in-code representation of a shape with textures. They can be obtained from multiple sources, for example by calling `UnbakedModel#bake` (default model loader) or `IUnbakedGeometry#bake` ([custom model loaders][modelloader]). Some [block entity renderers][ber] also make use of baked models. There is no limit to how complex a model may be.
+
+## Methods of `BakedModel`
+
+### `getQuads`
+
+The most important method of a baked model is `getQuads`. This method is responsible for returning a list of `BakedQuad`s, which can then be sent to the GPU. A quad compares to a triangle in a modeling program (and in most other games), however due to Minecraft's general focus on squares, the developers elected to use quads (4 vertices) instead of triangles (3 vertices) for rendering in Minecraft. `getQuads` has five parameters that can be used:
+
+- A `BlockState`: The [blockstate] being rendered. May be null, indicating that an item is being rendered.
+- A `Direction`: The direction of the face being culled against. May be null, indicating that no culling should occur. Will always be null for items.
```

Comment:
Items go through all possible directions (and null) just like blocks (see `ItemRenderer#renderModelLists`), so the direction is not always null for items.